doc/modules/ROOT/pages/examples.adoc

```diff
@@ -0,0 +1,46 @@
+= Examples
+James Elliott <james@deepsymmetry.org>; Simon Arnell <simon.arnell@configuredthings.com>; Mattias Holm
+
+The following are some examples of protocol contributed by the community.
```

Comment:
Protocols should be plural here, don’t you think?

Reply:
Corrected in 891ca58